src/tests/ftest/datamover/posix_types.py

```diff
@@ -185,6 +186,11 @@ def read_verify_location(self, path):
             param_type = 'POSIX'
             pool = None
             cont = None
+            self.log.info('--HERE-- verifying directory')
+            run_remote(self.log, self.hostlist_clients, 'cat /proc/mounts | grep mnt')
+            run_remote(self.log, self.hostlist_clients, 'df -h /mnt/share')
+            run_remote(self.log, self.hostlist_clients, 'ls -las "{}"'.format(path))
+            run_remote(self.log, self.hostlist_clients, 'ls -las $dirname("{}")'.format(path))
```

Comment:
I think I should have done
```suggestion
            run_remote(self.log, self.hostlist_clients, 'ls -las $(dirname "{}")'.format(path))
```